### DISCUSSION TITLE
SEO optimisation URLS / Title Tags

### DISCUSSION THREAD

**Discussion by SupaYoshi** [NamelessMC/Nameless#2500] [Ideas]
Implement URL optimisation and SEO optimisation in Custom Content pages, etc. for namelessmc,
To get websites using NamelessMC to rank better in search engines, URL optimisation is vital. along with H2, and title tags and short descriptions that are able to be set in the panel. Can this be implemented in the future?

More info on this:
SEO Guide to a Perfectly Optimized Landing Page
https://www.link-assistant.com/news/perfectly-optimized-page.html#anchor1

**Comment by samerton** [comment]
If you're using v2, you can add a description in the StaffCP -> Configuration -> SEO tab.

We do have h2 tags on several pages (including custom pages) in the Default Revamp template, there are likely a few more that need reviewing so if anyone spots any please do let us know.

Page titles can't be customised without editing code at the moment (outside of the `<page title> • <site name>` format which is in the template header.tpl file by default).
